experiment_name: "static_obstacles_scenario_report"
# if another world is used, add the world file to benchmarking_tool/world and provide the name of the new yaml file
world_name: 'scenario_world.world'
# if another map is used, add the map files to benchmarking_tool/map and provide the name of the new yaml file
map_name: 'map.yaml' 
# add gazebo model path, if multiple path, please spereate them by a :
models_path: ''

# Choose one of rhee following controllers, by adding its name to the controller type list

#DWB
#RPP
#RPP_RSC
#DWB_RSC

# if another controller is used, add the configuration file to turtlebot3_navigation2/param and provide the name of the new yaml file
controller_type: ['DWB','DWB_RSC', 'RPP']   #name of behaviour tree 
#Add the name of the robot navigation configuration, this file should be placed at turtlebot3/turtlebot3_navigation2
nav_config: 'burger'
controllers_num: 3
#provide the pose in a decimal form
spawn_pose_x: 0.0
spawn_pose_y: -3.4
spawn_pose_yaw: 1.58   #radians 


# Specify either to send one goal, specific trajectory, or several waypoints 
# square, circle, several_waypoints, one_goal
trajectory_type: 'one_goal'
# One goal parameter 
goal_pose_x: -1.3
goal_pose_y: 2.3
goal_pose_yaw: 1.58   #radians 

